code/src/components/Wrapper.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import Form from './Form'
+
+const Wrapper = () => {
```

Comment:
What is the difference between Form.js and Wrapper.js? I was thinking that it might be a good idea to add the Form.js directly into App.js and skip the wrapper component. But maybe there is a reason for it and that I don't have the knowledge to see it :)